### ISSUE TITLE
進捗

### ISSUE THREAD

**Issue by riron1206** [riron1206/kaggle_MoA#3]
#### 20201021
- lightGBMのコード修正中
    - モデルファイルuploadしてsubmissionできるように
    - cpu PCで学習実行試してる()

- label-smoothingのMLPのコード作成中
    - 自宅PCで学習実行中(C:\Users\81908\jupyter_notebook\poetry_work\tf23\01_MoA_compe\notebook\keras-multilabel-neural-network-v1-2)

--------------------------------------------------------
#### 20201022
- lightGBMのコード修正中
    - モデルファイルuploadしてsubmissionはできそう。まだ試せてない
    - Dataset uploadは1000ファイルが上限なので、cv=4でやるつもり（cv=5だと5*206=1030個モデルできるからダメだった。cv=5で学習一晩かかった）
    - optunaでパラメチューニングできるように修正中

- label-smoothingのMLPのsubmitした
    - https://www.kaggle.com/anonamename/submission-label-smoothing-20201021?scriptVersionId=45232016
    - public score: 0.01873
    - cv: コードミスで出せなかった（epochs=150で学習一晩かかった）
    - submit手順
        - Datasetにモデルファイルupload
        - kaggleでnotebook作成して実行

--------------------------------------------------------
